meson_options.txt

```diff
@@ -0,0 +1,21 @@
+option('packaging_backend', type : 'combo', choices : ['alpm', 'aptcc', 'dnf', 'dummy', 'entropy', 'pisi', 'poldek', 'portage', 'ports', 'slack', 'urpmi', 'yum', 'zypp', 'nix'], value : 'dummy', description : 'The name of the backend to use')
+option('dnf_vendor', type : 'combo', choices : ['fedora', 'mageia', 'openmandriva'], value : 'fedora', description : 'Vendor configuration when using the dnf backend')
```

Comment:
We're missing the newly-added `rosa` value here.

Reply:
Fixed, thanks. Will push after the next travis build.